src/jailer/src/main.rs

```diff
@@ -54,6 +54,7 @@ pub enum Error {
     MountBind(io::Error),
     MountPropagationSlave(io::Error),
     NotAFile(PathBuf),
+    NotAFolder(PathBuf),
```

Comment:
I think it is more appropriate to name this error `NotADir` or `NotADirectory`, because we refer to a filesystem path which represents a directory. The term `folder` is used when we refer to a directory, but as a GUI element.

Reply:
sure @iulianbarbu , will change Folder to Directory 👍 and add a unit test.